cli/sql_util_test.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2015 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License. See the AUTHORS file
+// for names of contributors.
+//
+// Author: Marc Berhault (peter@cockroachlabs.com)
+
+package cli
+
+import (
+	"bytes"
+	"database/sql"
+	"os"
+	"testing"
+
+	"github.com/cockroachdb/cockroach/server"
+)
+
+func makeTestDBClient(t *testing.T, s *server.TestServer) *sql.DB {
+	db, err := sql.Open("cockroach", "https://root@"+s.ServingAddr()+"?certs=test_certs")
+	if err != nil {
+		t.Fatal(err)
+	}
+	return db
+}
+
+func TestRunQuery(t *testing.T) {
```

Comment:
I really wanted to use the leaktest here, but the cli package imports os.signal which runs a signal loop at init time.